study.md

```diff
@@ -44,21 +45,22 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+<!-- A view is any piece of HTML that is generated but prompted to appear only after some user interaction. This HTML is sent from the server with loading of the original page, but does not appear until an action from the user causes it to.-->
 ```
```

Comment:
So the view is what you actually see on the scene. View state is referring to the data from the model that is being displayed to the user in the current view. So if we made an AJAX request and loaded data to our app, that data being displayed is the view state. Front end routers manage the view state based on interaction from the user.